contracts/MiniMeToken.sol

```diff
@@ -84,6 +84,7 @@ contract MiniMeToken is Controlled, IArbToken {
     // bytes32 public view VERSION_HASH =
     //      keccak256("1")
     bytes32 public constant VERSION_HASH = 0xc89efdaa54c0f20c7adf612882df0950f5a951637e0307cdcb4c672f298b8bc6;
+    address public constant BRIDGED_TOKENS_RESERVE = 0x0000000000000000000000000000000000B71d9E;
```

Comment:
Glad to see we were able to have the bridge as a configuration parameter. I wonder what was the reason to have an external address to hold the bridgeable tokens?

Reply:
So when users use the `approveAndCall()` function they can't pretend they're the owner of the tokens held in the MiniMeToken contract. External contracts will generally use the `receiveApproval()` `from` address but just to be sure. There might be other reasons I haven't thought of too and I don't think there's any advantage to keeping them in the MiniMeToken itself.